src/hooks/lastBlock/lib/getValidationData.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { ApiPromise } from '@polkadot/api';
+import { SignedBlock } from '@polkadot/types/interfaces';
+import { getValidationData, validationDataDataType } from './getValidationData';
+
+const rawValidationData = 'mockedRawValidationData';
+const mockJsonValidationData = 'result';
+const parachainBlock = {
+  block: { header: { hash: 'hash' } },
+} as unknown as SignedBlock;
+
+export const getMockApiPromise = (): ApiPromise =>
+  ({
+    query: {
+      parachainSystem: {
+        validationData: jest.fn(() => rawValidationData),
+      },
+    },
+    derive: {
+      chain: {
+        subscribeNewBlocks: jest.fn(),
+      },
+    },
+    createType: jest.fn(() => {
+      return {
+        isSome: true,
+        unwrap: () => ({ mockJsonValidationData }),
+      };
+    }),
+    at: jest.fn(() => getMockApiPromise()),
+  } as unknown as ApiPromise);
+
+describe('getValidationData', () => {
+  let mockApiInstance: ApiPromise;
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    mockApiInstance = getMockApiPromise();
+  });
+
+  it('gets validation data from parachain block head', async () => {
+    const jsonValidationData = await getValidationData(
+      mockApiInstance,
+      parachainBlock
+    );
+
+    expect(jsonValidationData).toEqual({ mockJsonValidationData });
```

Comment:
i'd prefer we'd also mock & check that the validation data is coming from the `.at` instance. Lets keep it as it is now, but down the line i'd rather test it like that